hack/build/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Pre-install packages
+FROM quay.io/centos/centos:latest
+
+# Install packages
+RUN yum update -y \
+    yum upgrade -y \
```

Comment:
Run a `yum clean all` https://stackoverflow.com/questions/46089219/how-to-reduce-the-size-of-rhel-centos-fedora-docker-image

Reply:
Can you also install python3 as well? We have intention to use that as a target backplane m3 language